tools/bazar/actions/BazarCartoAction.php

```diff
@@ -75,15 +75,18 @@ public function formatArguments($arg)
             // Zoom sur molette
             'zoom_molette' => $arg['zoommolette'] ?? $this->params->get('baz_wheel_zoom'),
             // Affichage en eclate des points superposes : true or false
-            'spider' => $arg['spider'] ?? 'false',
+            'spider' => $this->formatBoolean($arg, false, 'spider'),
             // Affichage en cluster : true or false
-            'cluster' => $arg['cluster'] ?? 'false',
+            'cluster' => $this->formatBoolean($arg, false, 'cluster'),
```

Comment:
```suggestion
            'cluster' => $arg['cluster'] ?? 'false',
```
même si ton bout de code par d'une bonne intention d'harmoniser les variables avec des booleans, celle-ci est un chaînes de caractère dans le template map (et dans les templates custom map), il ne faut donc pas le transformer en boolean à la rigueur
`'cluster' => $this->formatBoolean($arg, false, 'cluster') ? 'false' : 'true',`

Reply:
Yes bien vu ! comme je ne pense pas qu'un template map custom courre les rues, j'ai plutôt ajusté `map.tpl.html` pour gérer les vrais booléens. J'ai mis un "breaking-change" dans le commit pour ton changelog